src/gstools_cython/__init__.py

```diff
@@ -0,0 +1,33 @@
+"""
+Purpose
+=======
+
+GeoStatTools is a library providing geostatistical tools
```

Comment:
I just stumbled upon this. Has anyone ever used the name "GeoStatTools"? - Maybe we should start to only use "GSTools"?

Reply:
That is true. It's a remainder of the old times.

Reply:
Fixed